modules/tinymce/src/core/test/ts/browser/keyboard/FormatShortcutsTest.ts

```diff
@@ -0,0 +1,113 @@
+
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+import { describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks, TinySelections, TinyAssertions, TinyContentActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.keyboard.FormatShortcutsTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, []);
+
+  const platform = PlatformDetection.detect();
+
+  const testFormat = (editor: Editor, style: string) => {
+    TinyContentActions.keystroke(editor, style.charCodeAt(0), platform.os.isMacOS() ? { meta: true } : { ctrl: true } );
+  };
+
+  const Heading = (editor: Editor, number: string) => {
```

Comment:
As I said via DM's functions should always be camel case. So please rename this to something like `applyHeading`.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/e654525c48324835cdfc126db7bda677a061d1ce